artifactory_cleanup/artifactorycleanup.py

```diff
@@ -75,6 +82,9 @@ def cleanup(self, block_ctx_mgr, test_ctx_mgr) -> Iterator[CleanupSummary]:
                 print("Summary size not defined")
                 yield None
             print()
+            if self.save_removed_artifacts_list:
+                with open(os.path.join(self.save_removed_artifacts_path, f"{policy.name}.json"), 'w') as file:
```

Comment:
`policy.name` can contain unsupported characters by OS, we should escape it first